plasma_framework/contracts/src/exits/utils/ExitId.sol

```diff
@@ -2,31 +2,32 @@ pragma solidity 0.5.11;
 
 import "../../utils/Bits.sol";
 import "../../utils/PosLib.sol";
+import "./OutputId.sol";
 
 library ExitId {
     using PosLib for PosLib.Position;
-    using Bits for uint160;
+    using Bits for uint168;
     using Bits for uint256;
 
+    uint8 constant internal FIRST_BIT = 167;
+
     /**
      * @notice Checks whether exitId is a standard exit ID
      */
-    function isStandardExit(uint160 _exitId) internal pure returns (bool) {
-        return _exitId.getBit(151) == 0;
+    function isStandardExit(uint168 _exitId) internal pure returns (bool) {
+        return _exitId.getBit(FIRST_BIT) == 0;
     }
 
     /**
      * @notice Given transaction bytes and UTXO position, returns its exit ID
      * @dev Computation of a deposit ID is different to any other tx because txBytes of a deposit tx can be a non-unique value
-     * @notice Output index must be within range 0 - 255
      * @param _isDeposit Defines whether the tx for the exitId is a deposit tx
      * @param _txBytes Transaction bytes
      * @param _utxoPos UTXO position of the exiting output
      * @return _standardExitId Unique ID of the standard exit
      *     Anatomy of returned value, most significant bits first:
-     *     8-bits - output index
      *     1-bit - in-flight flag (0 for standard exit)
-     *     151-bits - hash(tx) or hash(tx|utxo) for deposit
+     *     167-bits - hash(tx) or hash(tx|utxo) for deposit
      */
     function getStandardExitId(
```

Comment:
minor: If you don't mind to sneak some small refactoring change, we can as well clean up the var name to be without underscore here.